### PR TITLE
Fix broken catalog-build-from-index command

### PR DESCRIPTION
**Pull request by sutaakar**
# Issue link

# What changes have been made

# Verification steps

## Checks
- [ ] I've made sure the tests are passing. 
- Testing Strategy
   - [ ] Unit tests
   - [ ] Manual tests
   - [ ] Testing is not required for this change

### CODE REVIEW

**Review by Bobbins228 — APPROVED** [review]
/lgtm

File: Makefile [hunk 1]
```diff
@@ -331,7 +331,7 @@ catalog-build-from-index: opm ## Build a catalog image.
 	mkdir catalog
 	$(OPM) render $(CATALOG_BASE_IMG) -o yaml > catalog/bundles.yaml
 	$(OPM) render $(BUNDLE_IMG) $(OPM_BUNDLE_OPT) > catalog/codeflare-operator-bundle.yaml
-	$(SED) -i -E "s/(.*)(- name: codeflare-operator.$(PREVIOUS_VERSION).*)/\1- name: codeflare-operator.$(VERSION)\n  replaces: codeflare-operator.$(PREVIOUS_VERSION)\n\2/" catalog/bundles.yaml
+	$(SED) -i -E "s/(.*)(- name: codeflare-operator.v0.2.0)/\1- name: codeflare-operator.$(VERSION)\n  replaces: codeflare-operator.$(PREVIOUS_VERSION)\n\2/" catalog/bundles.yaml
```
Comment (dimakis) [inline Makefile:334]: excuse my lack of sed knowledge here but are we now hardcoding a version? is this PR just for this fix?

**Review by dimakis — COMMENTED** [review]
just the one comment

File: Makefile (hunk 1 above)
Comment (sutaakar) [reply Makefile:334]: The purpose of this line is to inject replacing entry into the bundle section, where replacing entries are kept.
Originally this new replacing entry is placed right before last CodeFlare operator entry, however due to the regex issues with v1.0.0 the injection was duplicated. By fixing the searched text I made sure that only one injection is done.

Anyway this code is used just for tests, for real OLM release this bundle index is generated in OLM repo, we just provide manifests.

**Pull request merged**